app/settings.coverage.json

```diff
@@ -0,0 +1,13 @@
+
+{
+  "coverage": {
+    "coverage_app_folder": "/Users/Leo/WebstormProjects/winsleague/app/",
```

Comment:
On CI just use `spacejam --coverage` that setups automatically `coverage_app_folder` to the good folder and `is_active` to true. You should not use `--settings` while testing on CI platforms, like in this pull request